test/controllers/project_controller_test.rb

```diff
@@ -403,6 +403,35 @@ def test_add_member_member
     assert_equal(false, target_user.in_group?(eol_project.user_group.name))
   end
 
+  def test_add_member_writein
+    eol_project = projects(:eol_project)
+    target_user = dick
+    assert_not(target_user.in_group?(eol_project.admin_group.name))
+    assert_not(target_user.in_group?(eol_project.user_group.name))
+    params = {
+      id: eol_project.id,
+      candidate: "dick <Tricky Dick>"
```

Comment:
```suggestion
      candidate: "#{target_user.login} <#{target_user.name}>"
```
would make it easier to choose a different fixture if that were ever needed. I think also a bit less magical.